packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -1,15 +1,23 @@
+import { LionCalendar } from '@lion/calendar/src/LionCalendar';
 import { html, ifDefined, render } from '@lion/core';
 import { LionInputDate } from '@lion/input-date';
-import { withModalDialogConfig, OverlayMixin } from '@lion/overlays';
-
-import '@lion/calendar/lion-calendar.js';
-import './lion-calendar-overlay-frame.js';
+import { OverlayMixin, withModalDialogConfig } from '@lion/overlays';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { LionCalendarOverlayFrame } from './LionCalendarOverlayFrame.js';
 
 /**
  * @customElement lion-input-datepicker
  * @extends {LionInputDate}
  */
-export class LionInputDatepicker extends OverlayMixin(LionInputDate) {
+export class LionInputDatepicker extends ScopedElementsMixin(OverlayMixin(LionInputDate)) {
+  static get scopedElements() {
+    return {
+      ...super.scopedElements,
```

Comment:
`scopedElements` is static. Does this super call make sense?

Reply:
Yes, super refers to the parent constructor which can access static methods and properties.